docs/GettingStarted.rst

```diff
@@ -1361,6 +1363,18 @@ You can look for articles easily using the available search bar in the topbar of
 
 |image191|
 
+
+News application views
```

Comment:
I would use `filters` instead of `views`. WDYT ?

Reply:
I wrote Views to distinguish between filtering by spaces and the different views availables from the dropdown menu as it is not the same used component.

Reply:
Yes, but functionally speaking it is still filters. Talking about views make me think of different layouts for the News list.

Reply:
OK. I updated the text